CHANGES.rst

```diff
@@ -10,6 +10,7 @@ Changelog
 - #1628 Fix instrument import for analyses with result options
 - #1625 Fix assignment of analyses via worksheet template when Worksheet is full
 - #1617 Fix writing methods on read when reindexing services
+- #1633 Fix report section displays date pickers in Chinese
```

Comment:
@Espurna always add the changelog entry to the top of the list. They are always sorted from newest to oldest, as versions.